administrator/components/com_cookiemanager/forms/filter_cookies.xml

```diff
@@ -22,6 +22,7 @@
         name="category_id"
   			type="category"
   			label="JCATEGORY"
+        multiple="true"
```

Comment:
Please use tabs and not spaces to indent.

Reply:
I have indented XML files, is it correct now, please check?